bin/avg-tables-cell-label-analysis.R

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env Rscript 
```

Comment:
Sorry to be a nuisance guys- but should this maybe be a part of Andrey's Bioconda package? Will make it part of a container which will make life easier for us later.